test/it/base/IntegrationSpec.scala

```diff
@@ -9,14 +9,19 @@ import scala.reflect.ClassTag
 
 trait IntegrationSpec extends WordSpec with MustMatchers with ScalaFutures with IntegrationPatience with OptionValues {
 
+  val Port = 27017
+  val DatabaseName = "test-cds-file-upload"
+  val MongoURI = s"mongodb://localhost:$Port/$DatabaseName"
+  val CollectionName = "notifications"
+
```

Comment:
What do you think about extracting this part into separate `object` ? There is a pattern used in other services, for e.g. here: https://github.com/hmrc/customs-declare-exports-movements/blob/master/test/utils/stubs/TestMongoDB.scala